src/autoComplete/AutoComplete.story.mdx

```diff
@@ -0,0 +1,178 @@
+import { Meta, Preview, Props, Story } from '@storybook/addon-docs/blocks'
+
+import { Button, ButtonStatus } from '../button'
+import { SearchIcon } from '../icon'
+import { BaseSection } from '../layout/section/baseSection'
+import { AutoCompleteExample } from './AutoCompleteExample'
+import { AutoComplete } from './index'
+
+<Meta title="AutoComplete" />
+
+# **AutoComplete**
+
+### Default
+
+<Preview>
+  <Story name="Default">
+    <BaseSection>
```

Comment:
Please, avoid using `<BaseSection>` to stories if isn't mandatory to use it with this component

Reply:
Sadly it seems necessary